.eslintrc.js

```diff
@@ -75,6 +75,10 @@ module.exports = {
     'react/prop-types': 'off',
     'react/display-name': 'off',
     'react/state-in-constructor': 0,
+    'react/self-closing-comp': ['error', {
+      'component': true,
```

Comment:
since component/html=true is the default, you can omit them and just make the above line: `"react/self-closing-comp": "error"`

Reply:
good to know!